src/helpers.js

```diff
@@ -130,4 +130,20 @@ export function error(e, loader) {
 
 export function requiresMfa(e) {
 	return e.subcode && e.subcode.startsWith('MFA required');
+}
+
+export function getMfaStrategy(e) {
+	// Extract if it's authy or authenticator
+	const subcodeArray = e.subcode.split(':');
+	const authType = subcodeArray[1];
+
+	return {
+		mfaType: authType,
```

Comment:
Is the array order guaranteed?
There's no risk here that we return
`{ mfaType: 'AUTHY', hasAuthy: false }` when they have both?

Reply:
Yes, it's guaranteed. The API will return either:
- 'MFA required:AUTHY'
- 'MFA required:AUTHENTICATOR_APP'
- 'MFA required:AUTHENTICATOR_APP:hasAuthy'

Reply:
And yes, sorry the preference is always `'MFA required:AUTHENTICATOR_APP'` first if they have both

Reply:
You can [see here](https://github.com/raisely/raisely/blob/f5a8ec4e99621bc8489c9f82636ff6024868ff69/api/api/v3/controllers/mixins/authenticator.js#L181-L213) that these changes make the preferred app the Authenticator App, so it will throw that error first if the user has both.